.changes/unreleased/Features-20230620-224944.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Features
+body: Add support for dynamic tables
```

Comment:
```suggestion
body: Add support for dynamic tables, the Snowflake implementation of materialized views
```
or something like that 🤷‍♀️

Reply:
Snowflake also has materialized views. There is a desire to not use "materialized view" in any of the user facing components in order to reduce confusion.

Reply:
Oh? Interesting. Makes sense to leave as is